google/cloud/forseti/services/scanner/dao.py

```diff
@@ -350,9 +352,13 @@ def map_by_resource(violation_rows):
             v_data['resource_data'] = json.loads(
                 json.dumps(v_data['resource_data']))
 
-        v_resource = vm.VIOLATION_RESOURCES.get(v_data['violation_type'])
-        if v_resource:
-            v_by_type[v_resource].append(v_data)
+        violation_type = vm.VIOLATION_RESOURCES.get(v_data['violation_type'])
+        if not violation_type:
```

Comment:
Main fix. If violation type not found in the map, then check if the violation type starts with `cv_` and map violation to the CV violation type that Notifier is expecting.